### DISCUSSION TITLE
Getting JSON information in the overlaty

### DISCUSSION THREAD

**Discussion by panik988** [aaronwmorris/indi-allsky#1685] [Q&A]
Hello everybody
Is there a way to get json information from the weatherstation in the allsky overlay?
Thank you!

**Comment by aaronwmorris** [comment]
There is a setting in the Overlays tab for `Extra Image Text File` which can be used to add additional information to the overlay.  It follows all of the same rules as the other image labels.

https://github.com/aaronwmorris/indi-allsky/wiki/Image-Labels

If you have data that is being imported from OpenWeatherMaps or WeatherUnderground, those variables are available as the `sensor_user_#` variables referenced in the above page.

**Reply by panik988** [reply]
Thank you @aaronwmorris for your reply.
I'm just using that field for mlCloudDetect but my question was if I can read a JSON from a url path like 192.168.21.60/rd to get my weather station info.
WeatherUnderground is very usefull but it doesn't read SQM information.
Thank you
Marco

**Reply by aaronwmorris** [reply]
You would have to make an intermediate script to query the weatherstation and combine that with the mlCloudDetect output.
